src/components/WalletDropdown/MiniPortfolio/PortfolioLogo.tsx

```diff
@@ -57,34 +57,23 @@ const ENSAvatarImg = styled.img`
   width: 40px;
 `
 
-const StyledChainLogo = styled.img<{ isSquare: boolean }>`
-  height: ${({ isSquare }) => (isSquare ? '16px' : '14px')};
-  width: ${({ isSquare }) => (isSquare ? '16px' : '14px')};
-  margin-top: ${({ isSquare }) => (isSquare ? '0px' : '1px')};
-  margin-left: ${({ isSquare }) => (isSquare ? '0px' : '1px')};
-  position: absolute;
-  top: 68%;
-  left: 68%;
+const StyledChainLogo = styled.img<{ $fillParent: boolean }>`
+  height: ${({ $fillParent }) => ($fillParent ? '100%' : '14px')};
+  width: ${({ $fillParent }) => ($fillParent ? '100%' : '14px')};
```

Comment:
this is also confusing because layout should generally be determined by parent instead of the child component - see comment below about just making two separate components here